components/script/script_thread.rs

```diff
@@ -1590,7 +1590,7 @@ impl ScriptThread {
             let chan = &load.layout_chan;
             if chan.send(message::Msg::PrepareToExit(response_chan)).is_ok() {
                 debug!("shutting down layout for page {}", id);
-                response_port.recv().unwrap();
+                response_port.recv();
```

Comment:
I did not change the `let` code, i guess it was an error before i forked. Got fixed afterwards.